src/static/jakartaee-tutorial/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script type="text/javascript">
+window.location.replace('https://jakartaee.github.io/jakartaee-documentation/jakartaee-tutorial/current/');
```

Comment:
How does this compare to the meta [equiv approach](https://www.w3schools.com/tags/att_meta_http_equiv.asp)?

I can't think of why this wouldn't work, though. If they don't have JS enabled probably nothing will work anyway.

Should perhaps they be relative links?